test/runner.js

```diff
@@ -208,6 +208,29 @@ describe('Runner', function () {
         });
     });
 
+    it('debug domain error', function (done) {
+
+        var script = Lab.script();
+        script.experiment('test', function () {
+
+            script.test('a', function (finished) {
+
+                setImmediate(function () {
+
+                    throw new Error('throwing stack later');
+                });
+
+                finished();
+            });
+        });
+
+        Lab.execute(script, { D: true }, null, function (err, notebook) {
```

Comment:
Something strange I noticed. If I use `debug` insead of `D` like this:

```js
Lab.execute(script, { debug: true }, null, function (err, notebook) {
```

I get the console artifacts printed to the command line.